app/models/merchant.rb

```diff
@@ -11,4 +11,10 @@ def self.all_enabled
   def self.all_disabled
     where(status: 1)
   end
+
+  def items_ready_to_ship
+    items.joins(:invoices)
+    .where("invoices.status = ?", 0)
+    .select("items.*")
+  end
```

Comment:
I believe this model should be looking at the invoice_item status' (pending, packaged, shipped) instead of the invoice status. something like:

invoice_items.where(status: [0, 1])